packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3480,31 +3520,88 @@ class _ExitWidgetSelectionButtonGroupState extends State<_ExitWidgetSelectionBut
 
   bool _leftAligned = true;
 
+  Widget? get _moveExitWidgetSelectionButton {
+    final MoveExitWidgetSelectionButtonBuilder? buttonBuilder =
+        widget.moveExitWidgetSelectionButtonBuilder;
+    if (buttonBuilder == null) {
+      return null;
+    }
+
+    final String buttonLabel = 'Move to the ${_leftAligned ? 'right' : 'left'}';
+    return _TooltipGestureDetector(
+      button: buttonBuilder(
+        context,
+        onPressed: () {
+          _changeButtonGroupAlignment();
+          _onTooltipHidden();
+        },
+        semanticLabel: buttonLabel,
+        isLeftAligned: _leftAligned,
+      ),
+      onTooltipVisible: () {
+        _changeTooltipMessage(buttonLabel);
+      },
+      onTooltipHidden: _onTooltipHidden,
+    );
+  }
+
+  Widget get _exitWidgetSelectionButton {
```

Comment:
Can we make a helper widget class instead of a getter that returns a widget? same for _moveExitWidgetSelectionButton above

Reply:
These getters were added because without them conditionally building the widgets in the `build` method was a bit messy. They all use the same shared custom widget, and the arguments passed to the custom widget need to be defined on this parent widget (e.g. method for changing the tooltip, since the entire button group shares one tooltip), so adding a widget class for them I don't think would be much benefit. 

Note: I did rename the custom widget these getters use